package.json

```diff
@@ -49,14 +49,16 @@
     "jsdom": "^9.11.0",
     "lerna": "2.0.0-beta.38",
     "prettier": "^0.22.0",
+    "prop-types": "^15.5.6",
     "react": "^15.0.0-0",
     "react-addons-css-transition-group": "^15.0.0-0",
```

Comment:
This is deprecated, we need to switch to https://github.com/reactjs/react-transition-group instead.